### PR TITLE
changed nominal diameter to working diameter

### LINKED ISSUE #209: Rotational Frequency and Surface Velocity of Rolls Wrong When Pass Velocity Given
- Two Pi in `Roll.surface_velocity` Hook missing
- Connection between `Roll.surface_velocity` and different radii of roll unclear (working or max radius?)

### PR DESCRIPTION
**Pull request by ChRen95**
Closes #209.

### CODE REVIEW

**Review by axtimhaus — COMMENTED** [review]
The requested changes discussed in the issue regard the connection between rolling velocity (`RollPass.velocity`) and the velocity properties of the roll.

File: pyroll/core/roll/hookimpls.py [hunk 1]
```diff
@@ -31,7 +31,7 @@ def surface_velocity(self: Roll):
 
 @Roll.rotational_frequency
 def rotational_frequency(self: Roll):
-    return self.surface_velocity / self.nominal_radius
+    return self.surface_velocity / self.working_radius
```
[the remark this answers is not in the document]
Comment (ChRen95) [reply pyroll/core/roll/hookimpls.py:34]: Missing 2 pi is starting to be a thing. 
Yes you are right. But still, we have to discuss this as in plant automation they use the working diameter as well to set the rolls speed / frequency.

[the remark this answers is not in the document]
Comment (axtimhaus) [reply pyroll/core/roll/hookimpls.py:34]: Yes of course, but this is the **surface** velocity. Just do your thing in the hooks using `RollPass.velocity`

[the remark this answers is not in the document]
Comment (ChRen95) [reply pyroll/core/roll/hookimpls.py:34]: Okay, so to clearify that problem:
 - `RollPass.velocity` - The mean velocity of the Profile moving through the `RollPass` using the `Roll.working_radius`
 -  `RollPass.velocity` - The velocity at the neutral point using the `Roll.working_radius` (as of right now these two are equal)
 - Roll.surface_velocity - The `surface_velocity` of the `Roll` at the `nominal_radius`
 - Roll.working_surface_velocity - The `surface_velocity` of the roll at the `working_radius`

The problem arises when one wants to give the `Roll.rotational_frequency` witch then sets the different velocity values. When one takes this value from a given mill the `rotational_frequency` is set using the `working_radius` of a roll not it's nominal radius. Therefore `Roll.rotational_frequency` has to be linked to the `RollPass.velocity`.

**Comment by ChRen95** [comment]
@axtimhaus please review.

**Pull request merged**